add description for key update. (#109)
riscv-non-isa/riscv-ap-tee-io: specification/09-coveio_abi.adoc

| SBI_ERR_FAILED          | The operation failed for unknown reasons.
|===

[#sbi_covh_spdm_key_update]
===== Function: CoVE Host SPDM Key Update (FID #1033)
[source, C]
-----

> struct sbiret sbi_covh_spdm_key_update()(unsigned long device_id);

Reviewer: ```suggestion
struct sbiret sbi_covh_spdm_key_update(unsigned long device_id);
```

Author: done

-----

> Perform SPDM key update for a device.

Reviewer: ```suggestion 
Perform an SPDM key update for a the SPDM connection between the TSM and a connected device.
```

Author: done

The possible error codes returned in `sbiret.error` are shown below.

[#table_sbi_covh_spdm_key_update]
.CoVE Host SPDM Key Update
[cols="2,3", width=90%, align="center", options="header"]
|===
| Error code              | Description
| SBI_SUCCESS             | The operation completed successfully.
| SBI_ERR_FAILED          | The operation failed for unknown reasons.
|===

[#sbi_covh_ide_key_refresh]
===== Function: CoVE Host IDE Key Refresh (FID #1034)
[source, C]
-----

> struct sbiret sbi_covh_ide_key_refresh()(unsigned long device_id,

Reviewer: ```suggestion
struct sbiret sbi_covh_ide_key_refresh(unsigned long device_id,
```

Author: done

                                         unsigned long stream_id);
-----

> Perform IDE key refresh for a stream on a device.

Reviewer: ```suggestion
Perform an IDE key refresh for a connected device IDE stream.
```

Author: done
